### ISSUE TITLE
Hovering doesn't work when latent state is already "hover"

### ISSUE THREAD

**Issue by luke-hill** [mozilla/geckodriver#2173]
## System

* Version: 0.33
* Platform: Windows 10 / Ubuntu 2004
* Firefox: 121
* Selenium: Ruby 4.13+

## Testcase

(Cannot provide a minimal html file .... yet, this might not be possible as it's behind a login wall)
- Navigate using a link from page 1 to page 2
- Page 2 loads with a pseudo hover state already set (See image below)
- Attempt to hover on `Submissions v`
  - In firefox, the hover event fires, no error is generated, dropdown **is not** rendered
  - In chrome, the hover event fires, no error is generated, the dropdown **is** rendered

## Stacktrace

N/A - No error is generated

**Comment by luke-hill** [comment]
cc/ @whimboo

**Comment by whimboo** [comment]
Some questions...

1. What is a pseud-hover state already set and for which element / node?
2. What are you referring to with `Submission v`?
3. Could you try if the problem also exists when you save the affected page to disk? If yes, you could remove as much as possible content from the HTML/CSS/JS to get a minimized testcase?

Without a proper example it is very hard to assume what this problem is about exactly. :/

**Comment by luke-hill** [comment]
1) If you look at the image it looks like there is a hover state already set. This is because the page is probably coded poorly. There however is no hover state, this is the page's default state
2) The dropdown link called `Submissions` with a arrow down next to it.
3) I'll try that now. That's a solid suggestion. I'll update more in a day or two if I make any progress.

**Comment by luke-hill** [comment]
@whimboo - I've managed to download the webpage. And delete approx 50% of it. I've got it down to around 200 lines of code but there's still a lot I need to comb through. Once I have gotten the majority of sensitive items culled I'll upload a copy of it here or I can send it through on slack if easier

**Comment by whimboo** [comment]
That sounds great! And no worries if it takes a bit longer to remove all privacy related content. It's certainly still faster than trying to find such a test case out of the dark. Once you have the testcase maybe also include a small code snippet as well so that it's clear what needs to run. Thanks!

**Comment by luke-hill** [comment]
@whimboo - Ok here is what I have now (Sorry for delay)

HTML which is about 80 lines, it loads in the 2 css files locally.
css / application css files (I haven't tried to clean these up).

3 items total is 1.7meg. I can zip this up and try put it on in a few days once I've had someone just give it a sanitize check.

Is this enough compression?

**Comment by whimboo** [comment]
Yes, that sounds fine. I can trim it down further if needed. Important is that it works as a stand-alone testcase. Thanks!
